Adjust code samples for the confirm widget and added information about modalClass option

serhiyzhovnir opened the pull request:
## Purpose of this pull request

This pull request (PR) adds code samples for the confirm widget and also it adds the information about modalClass option.
The PR is an adjusted copy of https://github.com/magento/devdocs/pull/4732 PR.
I've created it on purpose to not to increase repo size too much.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_confirm.html
- https://devdocs.magento.com/guides/v2.2/javascript-dev-guide/widgets/widget_confirm.html
- https://devdocs.magento.com/guides/v2.1/javascript-dev-guide/widgets/widget_confirm.html

## Links to Magento source code

- https://github.com/magento/magento2/blob/2.1/app/code/Magento/Ui/view/base/web/js/modal/confirm.js#L17
- https://github.com/magento/magento2/blob/2.2/app/code/Magento/Ui/view/base/web/js/modal/confirm.js#L20
- https://github.com/magento/magento2/blob/2.3/app/code/Magento/Ui/view/base/web/js/modal/confirm.js#L20
- https://github.com/magento/magento2/blob/2.3/app/code/Magento/Checkout/view/frontend/web/js/sidebar.js#L90
- https://github.com/magento/magento2/blob/2.2/app/code/Magento/Checkout/view/frontend/web/js/sidebar.js#L90
- https://github.com/magento/magento2/blob/2.1/app/code/Magento/Checkout/view/frontend/web/js/sidebar.js#L74

devops-devdocs commented:
An admin must run tests on this PR before it can be merged.

jeff-matthews commented on guides/v2.1/javascript-dev-guide/widgets/widget_confirm.md:
```diff
@@ -109,18 +110,97 @@ The title of the confirmation window.
 **Default value**: `''`
 
+### `modalClass` {#confirm_modalClass}
+The CSS class of the confirm window.
```
Add space between header and paragraph

jeff-matthews commented on guides/v2.1/javascript-dev-guide/widgets/widget_confirm.md:
```diff
@@ -109,18 +110,97 @@ The title of the confirmation window.
 
 The keyboard navigation for the alert windows is similar to the [navigation of the modal widget].
 
+## Code Sample
```
Delete this line

jeff-matthews commented on guides/v2.1/javascript-dev-guide/widgets/widget_confirm.md:
```diff
@@ -109,18 +110,97 @@ The title of the confirmation window.
 
+## Code Sample
+
+### Code sample of initialization on an element
```
```suggestion
**Example 1**: initialization on an element
```

jeff-matthews commented on guides/v2.1/javascript-dev-guide/widgets/widget_confirm.md:
````diff
@@ -109,18 +110,97 @@ The title of the confirmation window.
+</script>
+```
+
+### Code sample of standalone initialization
````
```suggestion
**Example 2**: standalone initialization
```

rogyar commented:
Hi @jeff-matthews. Please, check the recent changes when you have time. Thank you!

jeff-matthews commented:
@rogyar & @atwixfirster, the original changes modified the 2.1 file, which was symlinked across versions. Since we've moved 2.1 files to a new branch and stopped accepting 2.1 PRs, this PR needs to be updated. After merging with `master` recently, there are no changed files associated with this PR.

These changes need to be applied to the 2.2 file.

rogyar commented:
Totally missed this point, thank you @jeff-matthews for the clarification. @serhiyzhovnir could you adjust your PR correspondingly, please?

serhiyzhovnir commented:
Hi @jeff-matthews @rogyar
I have noticed that the code examples for confirm widget are already added.
It was done in the scope of https://github.com/magento/devdocs/pull/4802 PR.
The examples of https://github.com/magento/devdocs/pull/4802 PR is really similar to this one.
We have adjusted the existed code examples and added information about the modalClass option.
Thank you!

jeff-matthews commented:
running tests
